src/main/scala/com/github/sbt/sbom/BomExtractor.scala

```diff
@@ -18,10 +19,27 @@ class BomExtractor(settings: BomExtractorParams, report: UpdateReport, log: Logg
     if (settings.includeBomSerialNumber && settings.schemaVersion != Version.VERSION_10) {
       bom.setSerialNumber(serialNumber)
     }
+    if (settings.schemaVersion.getVersion >= Version.VERSION_12.getVersion) {
+      bom.setMetadata(metadata)
+    }
     bom.setComponents(components.asJava)
     bom
   }
 
+  private def metadata: Metadata = {
+    val metadata = new Metadata()
+    metadata.addTool(tool)
+    metadata
+  }
+
+  private def tool: Tool = {
+    val tool = new Tool()
+    // https://github.com/devops-kung-fu/bomber searches for string CycloneDX to detect format
```

Comment:
weird so good to know :). a more detailed reference would be even nicer if you have one.